custom_components/airthings_wave/manifest.json

```diff
@@ -1,12 +1,12 @@
 {
   "domain": "airthings_wave",
   "name": "Airthings Wave",
-  "version": "3.1.0",
+  "version": "3.2.0",
```

Comment:
I'd increment this to 4.0.0 . It's a major upgrade ;)

Reply:
I fixed this, thanks for the feedback